### DISCUSSION TITLE
Blogpost #6

### DISCUSSION THREAD

**Discussion by philipsagawe** [lucariebel/online-shop#8] [General]
Diese Woche hat Sebastian die Oberfläche für "Auktion erstellen" erstellt:
![Screenshot 2024-11-18 at 16-04-38 FrontendRoot](https://github.com/user-attachments/assets/76e029e7-f931-493a-b37e-7a2f219b57e1)
<img width="1792" alt="image" src="https://github.com/user-attachments/assets/e26a3e51-ac32-4059-ba25-7989dcdf64be">
Des weiteren hat Oskar die Oberfläche für die Artikel auf der Hauptseite erstellt:
(zum jetzigen Zeitpunkt ist noch kein Screenshot vorhanden)
Luca hat das Speichern und Abrufen der Auktionen in der Datenbank realisiert.

**Comment by Mimikry2** [comment]
Moin,

Die Benutzeroberfläche für das Erstellen einer Auktion wirkt übersichtlich und benutzerfreundlich. Alle notwendigen Eingabefelder sind klar benannt und logisch angeordnet, was den Prozess für den Nutzer intuitiv macht. So wie es auf dem Screenshot aussieht kann man das Datum über einen Kalender-Picker auswählen, was sehr praktisch ist. Vielleicht wäre es sinnvoll, einen Mindest- und Maximalstartpreis festzulegen um unrealistische startpreise zu verhindern.

Viele Grüße,
Tim, Dennis, Nico

**Comment by GermanJesus-lul** [comment]
Hey,

Die Oberflächen sehen schonmal gut aus. Ist das Kategoriefeld ein Auswahl-/Drop-down-Feld und ist die Funktion des Einfügen von Bildern schon implementiert? Wie ist denn euer Backend bzw. eure gesamte Anwendung strukturell aufgebaut?

Mit freundlichen Grüßen,
Julius und das Rezeptbuch Team
